MaterialSkin/Controls/MaterialTabSelector.cs

```diff
@@ -58,7 +59,17 @@ public MaterialTabControl BaseTabControl
 
         private const int TAB_HEADER_PADDING = 24;
 
-        private const int TAB_INDICATOR_HEIGHT = 2;
+        private byte _tab_indicator_height;
+
+        [Category("Material Skin"), Browsable(true), DisplayName("Tab Indicator Height"), DefaultValue(2)]
+        public byte TabIndicatorHeight 
```

Comment:
Why byte? Int was good (although i doubt anyone will use an indicator bigger than 255px lol)

Reply:
Right, you can keep integer.